cli/lib/kontena/errors.rb

```diff
@@ -4,10 +4,33 @@ class StandardError < ::StandardError
 
       attr_reader :status
 
+      # @param status [Fixnum] HTTP response status
+      # @param message [String] short error message
       def initialize(status, message)
         @status = status
         super(message)
       end
     end
+
+    # The normal {error: {foo: "invalid foo"}} error response format used by the API
+    class StandardErrors < StandardError
```

Comment:
`StandardErrors` vs `StandardError` .. is that name a bit too close (makes spotting usage hard)?